eth-contracts/contracts/DelegateManager.sol

```diff
@@ -321,7 +321,7 @@ contract DelegateManager is InitializableV2 {
      * @dev Only callable by service provider. msg.sender is passed into processClaim
      * @dev Also factors in service provider rewards from delegator and transfers deployer cut
      */
-    function claimRewards() external {
+    function claimRewards(address _claimer) external {
```

Comment:
we could make the name more clear. this name is a bit verbose but i think it's much clearer _targetClaimAddress